server/ntb/mediatopics_to_subject_mapping.py

```diff
@@ -0,0 +1,50 @@
+import ntb
+
+from flask import current_app as app
+from superdesk import get_resource_service
+from superdesk.cache import cache
+from superdesk.timer import timer
+from superdesk.signals import item_publish
+from ntb.mediatopics import get_mediatopics
+
+
+def populate_subject(sender, item, **kwargs) -> None:
+    topics = get_mediatopics(item)
+    if not topics:
+        return
+    with timer("mediatopics:mapping"):
+        mapping = _get_topics_mapping()
+    subjects = []
+    for topic in topics:
+        subject = mapping.get(topic["qcode"])
+        if subject:
+            subjects.append(subject)
+
+    for subject in subjects:
+        if not any(
+            s["qcode"] == subject["qcode"] and s.get("scheme") == subject.get("scheme")
+            for s in item["subject"]
+        ):
+            item["subject"].append(subject)
+
+
+# @cache()
```

Comment:
It is commented, I think you forget to uncomment it?

Reply:
no I wanted to keep it commented for now

Reply:
will remove it for now